test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -62,6 +62,39 @@ class SolidityExecutionFramework: public solidity::test::ExecutionFramework
 	) override
 	{
 		bytes bytecode = multiSourceCompileContract(_sourceCode, _sourceName, _contractName, _libraryAddresses);
+
+		// Note: Temporary solution that adds arguments size in the EOF data_section_size header to validate evmone implementation.
+		// TODO: Validate header and version.
+		if (bytecode.size() > 3)
+		{
+			if (bytecode[0] == uint8_t(0xef) && bytecode[1] == uint8_t(00)) // is EOF
```

Comment:
```suggestion
			if (bytecode[0] == uint8_t(0xef) && bytecode[1] == uint8_t(0x00)) // is EOF
```
Is this a typo? I mean, they're equivalent anyway, but whatever :D